docs/using-latest-daily.md

```diff
@@ -54,6 +54,9 @@ Alternatively, to create a .NET Aspire project using the Starter template:
 dotnet new aspire-starter
 ```
 
+> [!TIP]
+> If you get an error saying `Unable to resolve the template, the following installed templates are conflicting`, append a `-9` to the above template names. For example, `dotnet new aspire-starter-9`.
```

Comment:
@joperezr @radical - do you think I should just use `-9` above in the commands? Anyone dogfooding likely has the Aspire 8 workload also installed.

Reply:
Given where we are trying to go, I think I should leave it as-is. The vision is for `dotnet new aspire-starter` to just work.